<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_QTopMenuButton</class>
 <widget class="QWidget" name="Test_QTopMenuButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>718</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="14" column="1">
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QGridLayout" name="gridLayout_8">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QRadioButton" name="labelShort">
        <property name="text">
         <string>Short</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="labelLong">
        <property name="text">
         <string>Long label button</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="labelEmpty">
        <property name="text">
         <string>&quot;&quot;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="labelVeryLong">
        <property name="text">
         <string>Very really long label for button</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Label:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QRadioButton" name="cellSide10">
        <property name="text">
         <string>10x10</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="cellSide30">
        <property name="text">
         <string>30x30</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="cellSide20">
        <property name="text">
         <string>20x20</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cell side size:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="19">
    <widget class="QWidget" name="Content" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QRadioButton" name="margin2">
        <property name="text">
         <string>2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="margin0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="margin8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Margin:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Enabled:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="enabledTrue">
        <property name="text">
         <string>true</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="enabledFalse">
        <property name="text">
         <string>false</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QRadioButton" name="Horizontal">
        <property name="text">
         <string>Horizontal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="Vertical">
        <property name="text">
         <string>Vertical</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Direction:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QRadioButton" name="sizeHint2x2">
        <property name="text">
         <string>45x45</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="sizeHint1x1">
        <property name="text">
         <string>20x20</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>SizeHints: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="sizeHint3x3">
        <property name="text">
         <string>70x70</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="sizeHint5x1">
        <property name="text">
         <string>120x20</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QRadioButton" name="sizeHint10x1">
        <property name="text">
         <string>240x20</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Content</tabstop>
  <tabstop>Horizontal</tabstop>
  <tabstop>Vertical</tabstop>
  <tabstop>sizeHint1x1</tabstop>
  <tabstop>sizeHint2x2</tabstop>
  <tabstop>sizeHint3x3</tabstop>
  <tabstop>sizeHint5x1</tabstop>
  <tabstop>sizeHint10x1</tabstop>
  <tabstop>cellSide10</tabstop>
  <tabstop>cellSide20</tabstop>
  <tabstop>cellSide30</tabstop>
  <tabstop>margin0</tabstop>
  <tabstop>margin2</tabstop>
  <tabstop>margin8</tabstop>
  <tabstop>labelEmpty</tabstop>
  <tabstop>labelShort</tabstop>
  <tabstop>labelLong</tabstop>
  <tabstop>labelVeryLong</tabstop>
  <tabstop>enabledTrue</tabstop>
  <tabstop>enabledFalse</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
